### ISSUE TITLE
[BUG] Crashes on OSX due to cpu[temp_high] syntax error: invalid arithmetic operato

### ISSUE THREAD

**Issue by ardunn** [aristocratos/bashtop#166]
**Describe the bug**

Crashes on OSX after a few minutes. Happens quite reliably on my system (i.e., every time bashtop is run).

Checking the `$HOME/.config/bashtop/error.log`, it is filled with the same kind of error:

```
New instance of bashtop version: 0.9.21 Pid: 51564
/Users/ardunn/alex/lbl/projects/common_env/dev_codes/bashtop/bashtop: line 2149: (7.1  11.41  10-20)*100/(cpu[temp_high]-20) : syntax error: invalid arithmetic operator (error token is ".1  11.41  10-20)*100/(cpu[temp_high]-20) ")

New instance of bashtop version: 0.9.21 Pid: 52804
/Users/ardunn/alex/lbl/projects/common_env/dev_codes/bashtop/bashtop: line 2149: (12.72  8.45  8-20)*100/(cpu[temp_high]-20) : syntax error: invalid arithmetic operator (error token is ".72  8.45  8-20)*100/(cpu[temp_high]-20) ")

New instance of bashtop version: 0.9.21 Pid: 53188
/Users/ardunn/alex/lbl/projects/common_env/dev_codes/bashtop/bashtop: line 2149: (16.96  10.75  9-20)*100/(cpu[temp_high]-20) : syntax error: invalid arithmetic operator (error token is ".96  10.75  9-20)*100/(cpu[temp_high]-20) ")

New instance of bashtop version: 0.9.21 Pid: 57856
/Users/ardunn/alex/lbl/projects/common_env/dev_codes/bashtop/bashtop: line 2149: (8.47  6.45  5-20)*100/(cpu[temp_high]-20) : syntax error: invalid arithmetic operator (error token is ".47  6.45  5-20)*100/(cpu[temp_high]-20) ")
```

**To Reproduce**

Installation of depedencies was exactly as under the README section `Dependencies installation OSX`

Then:

`git clone https://github.com/aristocratos/bashtop`
`cd bashtop`
`source bashtop`

**Expected behavior**

Bashtop should not crash.

**Screenshots**

N/a

**Info (please complete the following information):**
 - Bashtop version: 0.9.21
 - (OSX/FreeBSD) Os release version: Catalina 10.15.4
 - Terminal used: iTerm 3.3.12
 - Font used: Ubuntu Mono
 - Bash version, "bash --version" (version 4.4 or above is required): 5.0.17(1)-release (x86_64-apple-darwin19.4.0)
 - Locales: "en_US.UTF-8"

**Additional context**

N/a, error.log is already given above

**Comment by aristocratos** [comment]
What output do you get when you run `osx-cpu-temp`?

It's supposed to give one temperature for the cpu as a whole, but seems to get an array of 3 items:
` line 2149: (12.72  8.45  8-20)`

Only the last value `8` has it's decimal stripped since it was expecting 1 value, the `-20` is added by bashtop for normalization.
But the values you're getting seems strange too `12.72` Celsuis, unless you're cooling with liquid nitrogen!?! :)

**Comment by ardunn** [comment]
```
ardunn@eos [~]: osx-cpu-temp
51.1°C
```

**Comment by ardunn** [comment]
Actually, I think this may be a non issue. When I run `source bashtop` as opposed to `./bashtop`, it seems to run ok. But this issue might still be of interest to @aristocratos for bug fixing purposes

**Comment by ardunn** [comment]
@aristocratos feel free to close if you don't think this is worth investigating :)

Great tool by the way

**Comment by aristocratos** [comment]
Yeah, looks like the issue is caused by osx-cpu-temp and it haven't been updated in a while so I would suggest you make a bug report at https://github.com/lavoiesl/osx-cpu-temp if the problem persists. Could be some compatibility issue with OsX Catalina.

Will close this for now since I don't have a way of reproducing the problem.

And thanks :)

**Comment by aristocratos** [comment]
@ardunn 
Fixed in v0.9.24
